geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/BackCompatibilityTest.scala

```diff
@@ -49,7 +49,7 @@ class BackCompatibilityTest extends Specification with LazyLogging {
 
   sequential
 
-  lazy val connector = MiniCluster.connector
+  lazy val connector = MiniCluster.cluster.getConnector(Users.root.name, Users.root.password)
```

Comment:
Would it be better to use the ds.connector for this test?  (I don't know if that's a good idea or not.)

Reply:
yeah that probably makes sense. I'll update it

Reply:
fixed